File: Day71/Part3/combo_reforge.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    string s;
    if (cin >> s) {
        if (s.empty()) {
            cout << 0 << endl;
            return 0;
        }
        char target = s.back();
        int count = 0;
        for (char c : s) {
            if (c != target) {
                count++;
            }
        }
        cout << count << endl;
    }
    return 0;
}

File: Day71/Part3/combo_reforge.py
def solve():
    try:
        s = input().strip()
        if not s:
            print(0)
            return
            
        target = s[-1]
        count = 0
        for c in s:
            if c != target:
                count += 1
                
        print(count)
            
    except EOFError:
        pass

if __name__ == "__main__":
    solve()
